contracts/deployments/default.simnet-plan.yaml

```diff
@@ -7,18 +7,23 @@ genesis:
     - name: deployer
       address: ST2SBXRBJJTH7GV5J93HJ62W2NRRQ46XYBK92Y039
       balance: "100000000000000"
+      sbtc-balance: "1000000000"
```

Comment:
This is a new default setting in Clarinet, that is automatically set. Can be ignored